File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_vb_exercises/App_Code/TrainingV1/TrainingCourse.vb
Imports Microsoft.VisualBasic

Namespace TrainingV1
    Public Class TrainingCourse
        Public Function GetDelegates() As List(Of TrainingDelegate)
            Dim attendees As List(Of TrainingDelegate) = New List(Of TrainingDelegate)()
            attendees.Add(New TrainingDelegate("Joe Bloggs", "BMW", 3, 2500.0))
            attendees.Add(New TrainingDelegate("Jane Smith", "Honda", 5, 2600.0))
            attendees.Add(New TrainingDelegate("Fred Martin", "Nissan", 2, 2700.0))
            attendees.Add(New TrainingDelegate("Sarah Jenkins", "Peugeot", 1, 2800.0))
            attendees.Add(New TrainingDelegate("Peter Griffiths", "Renault", 7, 2900.0))
            Return attendees
        End Function
    End Class
End Namespace

File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_cs_exercises/App_Code/AspxExercises/TrainingV1/TrainingCourse.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Configuration;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;

namespace AspxExercises.TrainingV1 {
    public class TrainingCourse {
        public TrainingCourse() {
        }
        public List<Delegate> GetDelegates() {
            List<Delegate> attendees = new List<Delegate>();
            attendees.Add(new Delegate("Joe Bloggs", "BMW", 3, 2500.0));
            attendees.Add(new Delegate("Jane Smith", "Honda", 5, 2600.0));
            attendees.Add(new Delegate("Fred Martin", "Nissan", 2, 2700.0));
            attendees.Add(new Delegate("Sarah Jenkins", "Peugeot", 1, 2800.0));
            attendees.Add(new Delegate("Peter Griffiths", "Renault", 7, 2900.0));
            return attendees;
        }
    }
}
